helm/ovn-kubernetes/charts/ovnkube-control-plane/templates/rbac-ovnkube-cluster-manager.yaml

```
@@ -88,9 +90,15 @@ rules:
           - clusteruserdefinednetworks
           - clusteruserdefinednetworks/status
           - clusteruserdefinednetworks/finalizers
+          - uplinks
+          - uplinks/status
           - vteps
           - vteps/status
       verbs: [ "patch", "update" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)